wlauto/utils/statedetect.py

```diff
@@ -91,14 +91,24 @@ def match_state(screenshot_file, defpath, state_definitions):  # pylint: disable
     for template_png in template_list:
         template = cv2.imread(os.path.join(defpath, 'templates', template_png + '.png'), 0)
         template_edge = auto_canny(template)
+        (tH, tW) = template_edge.shape[:2]
```

Comment:
Names should be more descriptive and all lower case: `template_height` and `template_width`